### DISCUSSION TITLE
Almgren-Chriss additive formula for base fee update

### DISCUSSION THREAD

**Discussion by mtefagh** [filecoin-project/FIPs#231] [Enhancements - Technical]
Migrated from [#92](https://github.com/filecoin-project/FIPs/issues/92); originally authored by @mtefagh

---
Recent [empirical studies](https://arxiv.org/abs/2110.04753) have shown that EIP-1559 has caused “intense, chaotic oscillations in block sizes.” I predicted this behavior more than two years ago [here](https://ethereum-magicians.org/t/eip-1559-fee-market-change-for-eth-1-0-chain/2783/24?u=mtefagh). Recently, I did prove some theoretical results about why this is the case in [here](https://ethresear.ch/t/path-dependence-of-eip-1559-and-the-simulation-of-the-resulting-permanent-loss/8964?u=mtefagh). And I have elaborated on the different aspects of my proposed solutions [here](https://hackmd.io/@PIncentivus/EIP-3416).

However, if you don't have time to go over all the materials which I have sent above, just read the following lines:
If the fee update formula is multiplicative like in EIP-1559 and one block is 10% underused and the next block is 10% overused, the overall net effect of decreasing the base fee by 10% and then increasing it by 10% would be reducing the base fee by one percent in total (0.9*1.1 = 0.99). This can be shown that any other kind of oscillation is also incentivized in the same way. Moreover, because there is no slippage in EIP-1559 and the transactions in one block only impact the fee price in the next block, this incentivizes dumping all one's transactions at once instead of sending them gradually, which is yet another reason for fluctuations. For another simple example, if the current block is overused, not only you are not incentivized to wait as you know for sure that the fee price will increase for inclusion in the next block, but also you are incentivized to get on this overused block and make it even more full as you know there is no price impact for your own transactions because of this one block delay in EIP-1559.

**Comment by kaitlin-beegle** [comment]
FYI - new staff have onboarded to Protocol Labs' CryptoEconLab and are reviewing this proposal.
